examples/salesforce-lightning-design-system/webpack.config.js

```diff
@@ -58,5 +58,8 @@ module.exports = {
         contentBase: path.resolve(__dirname),
         port: 8888,
         inline: false,
+        headers: {
+            "Access-Control-Allow-Origin": "*",
+        },
```

Comment:
This actually needs to change in the quip repo, stuff in `examples` is periodically overwritten from there.

Reply:
@jesseditson  changed it in the quip repo in https://github.com/quip/quip/pull/11342
should I still change it here too? what's the process of copying the examples over?

Reply:
You can run scripts/sync_example_live_apps.sh to update these - you can pass the path to this repo as an argument and commit the changes directly to master (or PRing them is fine too)